From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 10 Aug 2016 16:43:39 +0200
Subject: [PATCH] build: Allow PLUGINS override

Instead of using BUILD_FUEL_PLUGINS, which was intended for manual
overrides of the Fule@OPNFV plugin list, allow PLUGINS to be
preset to a custom list, which we will provide via Armband
Makefile spec.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 build/f_isoroot/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/f_isoroot/Makefile b/build/f_isoroot/Makefile
index 4df7bcb..448f415 100644
--- a/build/f_isoroot/Makefile
+++ b/build/f_isoroot/Makefile
@@ -11,7 +11,7 @@
 # Add Fuel plugin build targets here
 # Plugins disabled for the Fuel Mitaka uplift. Please re-enable your plugin as you have
 # verified it!
-PLUGINS = f_odlpluginbuild f_onosfwpluginbuild f_vsperfpluginbuild f_ovs-nsh-dpdk-pluginbuild f_yardstick-pluginbuild f_kvm-pluginbuild f_bgpvpn-pluginbuild f_collectd-ceilometer-pluginbuild
+PLUGINS ?= f_odlpluginbuild f_onosfwpluginbuild f_vsperfpluginbuild f_ovs-nsh-dpdk-pluginbuild f_yardstick-pluginbuild f_kvm-pluginbuild f_bgpvpn-pluginbuild f_collectd-ceilometer-pluginbuild
 #PLUGINS = f_odlpluginbuild f_onosfwpluginbuild f_ovsnfv-dpdk-pluginbuild f_vsperfpluginbuild f_ovs-nsh-dpdk-pluginbuild f_bgpvpn-pluginbuild
 export PLUGINS
 #NON_8-0_REBASED_PLUGINS = f_bgpvpn-pluginbuild
